### ISSUE TITLE
开启数据流之后，CPU占用很高

### ISSUE THREAD

**Issue by sqqs9490** [kingToolbox/WindTerm#648]
![image](https://user-images.githubusercontent.com/37857095/162693862-819fa476-d939-414e-afee-34a304ab2146.png)

**Comment by kingToolbox** [comment]
Yes, because when `WindTerm` receives data, it does many things more than other terminals, including `Word Wrap`, `Styling`, `Folding`, `Outlining` and `Compression` etc. Especially `Word wrap`, may consume more CPU. You can try turning off `Word Wrap` (`Menu - Mode - Word Wrap`, `Alt+M, Alt+W`) and try again.

I will try to improve the performance of `Word Wrap` in subsequent releases. If there is any progress I will update here. Please stay tuned, thank you.

**Comment by standback** [comment]
is there a way to disable  all kinds of  txt process. 
the cpu usage is too high for continuous output.

i think it is better to provide the option to user,  to decide what really need.
